### ISSUE TITLE
Canvas does not work in Virtual Box and Real hardware.

### ISSUE THREAD

**Issue by MishaTY** [CosmosOS/Cosmos#1580]
#### Area of Cosmos - What area of Cosmos are we dealing with?
CGS

#### Expected Behaviour - What do you think that should happen?
OS should display colors/text correctly

#### Actual Behaviour - What unexpectedly happens?
My thinkpad laptop colors are off. Virtual box does not display anything at all.

#### Reproduction - How did you get this error to appear?
Used this code: https://pastebin.com/gNK7jSek

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
Dev kit. #1573
#### Screen Shots
Real hardware:
![WIN_20201201_10_24_11_Pro](https://user-images.githubusercontent.com/46088515/100760487-cd3c8e00-33bf-11eb-9cef-54eaec3aa815.jpg)
Virtual box:
![rA3UGuSuoU](https://user-images.githubusercontent.com/46088515/100760559-e2192180-33bf-11eb-842c-0ae7af589ca1.png)

**Comment by Edx09** [comment]
Have you set up Virtualbox corectly?

**Comment by MishaTY** [comment]
Ok, nevermind about Virtualbox. I set the graphics controller to VMSVGA and it worked. But it is very slow.

**Comment by quajak** [comment]
For the real hardware can you test that no colors work, like test drawing a transition between red and green for example?

**Comment by MishaTY** [comment]
Ok, here is the 2nd test:

Virtual Machine:
![0Qghe8O0SV](https://user-images.githubusercontent.com/46088515/100795858-b90f8580-33ed-11eb-98bb-ae189f79605d.png)
Real hardware:
![WIN_20201201_15_55_07_Pro](https://user-images.githubusercontent.com/46088515/100795887-c3318400-33ed-11eb-8981-d9d756d375ac.jpg)

**Comment by Edx09** [comment]
Have you written your own display driver or are you ussing the cosmos one

**Comment by MishaTY** [comment]
I am using the cosmos one.

**Comment by Edx09** [comment]
well that could be the problem because it is written for vmware (i think)

**Comment by MishaTY** [comment]
No, I am talking about the FullscreenCanvas.GetFullscreenCanvas()
